src/MenuItem/MenuItem.js

```diff
@@ -59,6 +59,10 @@ class MenuItem extends Component {
 
   static propTypes = {
     /**
+     * Override the default anchorOrigin children used.
+     */
+    anchorOrigin: PropTypes.object,
```

Comment:
Please use the `utils/propTypes.origin` helper. You can have a look at the other components to see how it's done.